### PR TITLE
7017  update releasing 1.3 

### LINKED ISSUE #7017: releasing v1.3 -- checklist
tentative: ~6th Oct~ 12th Oct

## Review the deprecated APIs
- [x] remove any promised deprecated functionality (https://github.com/Project-MONAI/MONAI/issues/7018)

## Release a candidate version
- [x] Tag and release PYPI version [X.Y.Z]rc[K] following [the contributing guide](https://github.com/Project-MONAI/MONAI/blob/master/CONTRIBUTING.md#release-a-new-version).
- [x] Make sure all CI pipelines pass.

## Verify the new PyPI release
- [x] Check [the PyPI page](https://pypi.org/project/monai/0.2.0/) 
- [x] Test `pip install monai==[X.Y.Z]rc[K]` and run a few MONAI modules locally
- [x] Test all installation commands mentioned in [the installation guide](https://docs.monai.io/en/latest/installation.html) and [the primary README](https://github.com/Project-MONAI/MONAI/blob/master/README.md).

## Quality assurance for the relevant repos
The new PyPI release candidate should be checked against all the relevant repositories under [`Project-MONAI`](https://github.com/Project-MONAI), for example, [the tutorials repo](https://github.com/Project-MONAI/tutorials):
- [x] Run all Jupyter notebooks, replace `pip install monai` with `pip install monai==[X.Y.Z]rc[K]`.
- [x] File new tickets and create new pull requests to address any technical issues.
- [x] Release a new candidate after addressing all the issues.
- [x] Testing related repositories
  - [x] MONAI unit and integration tests
  - [x] MONAI tutorial tests
  - [x] MONAILabel unit and integration tests
  - [x] MONAI model zoo tests
  - [ ] QA regression tests

## Prepare documentation
- [x] Draft a release note [on GitHub](https://github.com/Project-MONAI/MONAI/releases).
- [x] Draft a release note with a pull request to modify [the CHANGELOG](https://github.com/Project-MONAI/MONAI/blob/master/CHANGELOG.md).
- [x] Create a pull request to highlight the features of the upcoming release, for example, in [`docs/source/highlights.md`](https://github.com/Project-MONAI/MONAI/blob/master/docs/source/highlights.md).
- [x] Create a pull request to update [the primary README.md](https://github.com/Project-MONAI/MONAI/blob/master/README.md).
- [x] Update the `monai.io` website (https://github.com/Project-MONAI/project-monai.github.io).
- [x] Check all the URLs in `https://monai.io` 
- [x] Check all the URLs in `https://docs.monai.io`.

## Release a milestone version
- [x] Tag and release PyPI version [X.Y.Z] following [the contributing guide](https://github.com/Project-MONAI/MONAI/blob/master/CONTRIBUTING.md#release-a-new-version).
- [x] Check the dockerhub auto-build by running `docker pull projectmonai/monai:X.Y.Z` and `docker run ...`
- [x] Check jupyter lab/notebook in the docker container
- [x] Check readme frontpage on dockerhub, pypi
- [x] Update https://github.com/Project-MONAI/MONAI/blob/dev/CITATION.cff
- [x] Publish the release note [on GitHub](https://github.com/Project-MONAI/MONAI/releases).
- [x] Update the weekly-preview tagging https://github.com/Project-MONAI/MONAI/blob/master/.github/workflows/weekly-preview.yml#L32
- [x] Release the corresponding conda-forge version https://github.com/conda-forge/monai-feedstock

## Revisit the GitHub project tab
- [ ] Update https://github.com/Project-MONAI/MONAI/projects.

### PR DESCRIPTION
**Pull request by wyli**
Fixes #7017

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: .github/workflows/weekly-preview.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ jobs:
       with:
         ref: dev
         fetch-depth: 0
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```
Comment (KumoLiu) [inline .github/workflows/weekly-preview.yml:16]: May I ask why we need to update to 3.9 here?
Do we also update here?
https://github.com/Project-MONAI/MONAI/blob/865972f7a791bf7b42efbcd87c8402bd865b329e/setup.cfg#L36

**Review by KumoLiu — APPROVED** [review]

File: .github/workflows/weekly-preview.yml (hunk 1 above)
Comment (wyli) [reply .github/workflows/weekly-preview.yml:16]: This is for dev preview, at some point next year (1.4 or 1.5) we'll remove the 3.8 support..

**Pull request merged**